_sass/main.scss

```diff
@@ -460,13 +460,25 @@ table code {
    h1, h2, h3, h4, h5 {
        font-family: "Titillium Web", sans-serif;
    }
-
 }
 
-.javadoc-page-title {
-   color: $title-colour;
-   font-size: 35px;
-   font-family: "Titillium Web", sans-serif;
+.documentation {
```

Comment:
To simplify things a bit, instead of adding a new block for `.documentation`, we can just add `.documentation` to the two blocks above so the font size, padding, and headings are consistent across the different pages.

Reply:
@fjuma due to the size do you think we can merge as is and then follow up with subsequent changes?

Reply:
@darranl Yes, this can be merged as is.